.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "10:00"
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: python-dotenv
+    versions:
+    - 0.15.0
+    - 0.16.0
+  - dependency-name: flask-jwt-extended
+    versions:
+    - 4.0.2
```

Comment:
@epicadk why is this being ignored? just want to make sure this makes sense, I am not sure?

Reply:
Probably because someone closed those prs. We could just remove it.

Reply:
This happens when PRs for those versions are closed or marked as ignore. `flask-jwt-extended` was manually upgraded by @epicadk . That's why depandabot ignores it :thinking:

Reply:
ok so if I remove this whole `ignore` section, will this config work @epicadk and @devkapilbansal ?

Reply:
Yes, the config should work

Reply:
> ok so if I remove this whole `ignore` section, will this config work @epicadk and @devkapilbansal ?

@isabelcosta can you do this so we can upgrade?